### PR TITLE
Fix Quoting of "joinTable" table names in ORMPurger

### PR DESCRIPTION
**Pull request by lavoiesl**
Fixes #139

### CODE REVIEW

**Comment by Ocramius** [comment]
@lavoiesl this needs an additional test case as well

**Comment by lavoiesl** [comment]
True that.

File: lib/Doctrine/Common/DataFixtures/Purger/ORMPurger.php [hunk 1]
```diff
@@ -185,17 +185,22 @@ private function getCommitOrder(EntityManagerInterface $em, array $classes)
         return $calc->getCommitOrder();
     }
 
-    private function getAssociationTables(array $classes)
+    /**
+     * @param array $classes
+     * @param \Doctrine\DBAL\Platforms\AbstractPlatform $platform
+     * @return array
+     */
+    private function getAssociationTables(array $classes, $platform)
```
Comment (stof) [inline lib/Doctrine/Common/DataFixtures/Purger/ORMPurger.php]: the argument should be typehinted

Comment (lavoiesl) [inline lib/Doctrine/Common/DataFixtures/Purger/ORMPurger.php]: fixed

**Pull request merged**